### PR TITLE
Implemented Card Component for caregroup 

### PR DESCRIPTION
**Pull request by evantaurusta**
- installed eslint, and react Material UI
- Axios simple GET

### CODE REVIEW

File: src/containers/Caregroup/Caregroup.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import CaregroupCard from '../../components/Card/caregroup-card';
```
Comment (sdkitono) [inline src/containers/Caregroup/Caregroup.js]: component naming to CareGroupCard

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_USER=techadmin
+REACT_APP_API_PW=d3vTEAMapi2018!
```
Comment (sdkitono) [inline .env:2]: Change this please PW and put this file into gitignore

**Review by sdkitono — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (evantaurusta) [reply .env:2]: done

**Review by sdkitono — DISMISSED** [review]

**Review by socraticat — CHANGES_REQUESTED** [review]
Pls remove all Contact Us pages/references. Will be replaced with Prayer Request.

**Review by socraticat — APPROVED** [review]

**Pull request merged**